tests/pytest.ini

```diff
@@ -1,2 +1,7 @@
 [pytest]
-log_level=INFO
+log_level=WARN
```

Comment:
Attempting to make the output from the system tests less verbose, as it there can be an unreal amount of scrolling up required to find the actual error. This helps some, but it seems the same logging is winding up on stderr which is getting captured and displayed by pytest as well. But now it's only printing it once instead of twice, which is an improvement.

Reply:
Appreciate this, agree the scrolling is maddening